CHANGELOG.md

```diff
@@ -6,9 +6,17 @@ The format is based on [Keep a Changelog]
 
 ## [Unreleased]
 
+## [Relase 048] - 2020-01-28
```

Comment:
Can we fix the typo ("Relase") in this and some previous ones?

Reply:
Ah, yep, it's typo'd for Release 47 as well.

Reply:
And 46! Damn copypastas